gencert-private.sh

```diff
@@ -20,17 +40,15 @@ agid=1.3.76.16
 spid-privatesector-SP=1.3.76.16.4.3.1
 spid-publicsector-SP=1.3.76.16.4.2.1
```

Comment:
This is the script for private sector. You don't need `1.3.76.16.4.2.1` (and relatives)